Microsoft.Toolkit.Uwp.UI.Controls/Eyedropper/Eyedropper.Logic.cs

```diff
@@ -50,9 +50,20 @@ private void UpadateWorkArea()
             else
             {
                 var left = WorkArea.Left;
-                var right = Window.Current.Bounds.Width - WorkArea.Right;
                 var top = WorkArea.Top;
-                var bottom = Window.Current.Bounds.Height - WorkArea.Bottom;
+                double right;
+                double bottom;
+                if (ControlHelpers.IsXamlRootAvailable && XamlRoot != null)
```

Comment:
We have this pattern a couple of times, here for window size, above for DPI Scale. Does it make sense to make a couple of general helpers for this type of common information which abstracts this away more easily? Maybe they can be extensions to UIElement called like GetParentContainerSize() and GetParentContainerDPI()? so it makes sense in either context?

Reply:
I don't think we should. These would be too simple, and they will go away on 8.0 since we'll be on a min version that this won't make sense any more.